### ISSUE TITLE
Error in example 5: HTTPError: 403 Client Error: Forbidden for url: https://huggingface.co/gpt2/resolve/main/vocab.json

### ISSUE THREAD

**Issue by Gasoonjia** [Adapter-Hub/adapter-transformers#213]
## Environment info

     
- `adapter-transformers` version:
- Platform: colab
- Python version:
- PyTorch version (GPU?):
- Tensorflow version (GPU?):
- Using GPU in script?:
- Using distributed or parallel set-up in script?:

## Information

Model I am using (Bert, XLNet ...): GPT2

Language I am using the model on (English, Chinese ...): English

Adapter setup I am using (if any): no

The problem arises when using:
* [ ✔️] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [ ] my own task or dataset: (give details below)

## To reproduce

1. Run example 05_Text_Generation.ipynb
2. The error is shown in the third block.
3.

```
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
<ipython-input-3-9640d43b7684> in <module>()
      8   return encoding
      9 
---> 10 tokenizer = GPT2Tokenizer.from_pretrained("gpt2")
     11 # The GPT-2 tokenizer does not have a padding token. In order to process the data
     12 # in batches we set one here

4 frames
/usr/local/lib/python3.7/dist-packages/requests/models.py in raise_for_status(self)
    939 
    940         if http_error_msg:
--> 941             raise HTTPError(http_error_msg, response=self)
    942 
    943     def close(self):

HTTPError: 403 Client Error: Forbidden for url: https://huggingface.co/gpt2/resolve/main/vocab.json

```

**Comment by calpt** [comment]
Hi @Gasoonjia,

we just released a fix for this with `adapter-transformers` v2.1.2. Please verify if this resolves your issue. Thanks!

**Comment by Gasoonjia** [comment]
It works! Thanks.
